src/routing_node.rs

```diff
@@ -64,6 +65,7 @@ pub struct RoutingNode<F: Facade> {
     accepting_on: Option<u16>,
     next_message_id: MessageId,
     bootstrap_node_id: Option<DhtId>,
+    filter: LruCache<types::FilterType, ()>,
```

Comment:
yes, agreed, is add_key already implemented in message_filter?  I figured first put in lru_time_cache and then replace it with message filter.  Guess i lost too much time doing nothing over the weekend :)  I'll update it